admin_tools_stats/templates/admin_tools_stats/chart_form.html

```diff
@@ -18,18 +19,22 @@
                 no-repeat;
     }
 </style>
-<form class="stateform" action="." method="POST" enctype="multipart/form-data">
-   <input id="load_on_change" title="Load on change" checked type="checkbox"/>⇒&nbsp;
+<form class="stateform form" action="." method="POST" enctype="multipart/form-data">
+   <input id="load_on_change" title="Load on change" checked type="checkbox"/> ⇒ &nbsp;
    {% csrf_token %}
    {% for w in form %}
       {% if not w.is_hidden %}
       {{ w.label }}:
       {% endif %}
       {{ w }}
    {% endfor %}
-   &nbsp;<button class="reload" id="reload" title="reload values from unfinished periods" type="button" style="font-size:14px">⟳</button>
+   <br>
```

Comment:
Putting the reload button on newline might not be always the best option. For example if I have charts in two columns, it would result in this less compact layout with 3 row controls:

![Snímek obrazovky_2024-04-15_09-59-21](https://github.com/PetrDlouhy/django-admin-charts/assets/156755/dbbfa103-916f-47b1-ab68-204b14115315)

I don't think, that the controls layout is optimal, but I am not sure, if this change will make it better.

Also, if we are new line, you should remove the `&nbsp;` from the beginning of the line to align all elements at the start.